.github/workflows/my-ci-pipeline.yml

```diff
@@ -14,3 +14,21 @@ jobs:
     - uses: actions/checkout@v4
     - run: docker build --tag todo-app:test --target test .
     - run: docker run --env-file .env.test todo-app:test
+
+  deploy:
+    name: Push to Docker Hub trigger app service deploy 
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/exercise-9' || github.ref == 'refs/heads/main' }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo hello deploy!
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{secrets.DOCKER_USERNAME}}
+          password: ${{secrets.DOCKER_TOKEN}}
+      - run: docker build --tag cristianodeoliveira/todo-app:prodlatest --target production .
```

Comment:
`prodlatest` is a slightly odd tag here, not that there's anything wrong with it but usually I'd expect either `prod` or `latest` depending on how we want to manage our images